lib/services/devices/deviceService.js

```diff
@@ -318,8 +318,14 @@ function createInitialEntityNgsi2(deviceData, newDevice, callback) {
  * @param {Object} newDevice        Device object that will be stored in the database.
  */
 function createInitialEntityNgsi1(deviceData, newDevice, callback) {
```

Comment:
No CNR entry needed? Is related with something already include in CNR?

Reply:
This is related to `Fix: use trust and cbHost from deviceGroup (#685)`. I think that we don't need to add a new netry.

Reply:
Ok. NTC